src/pages/Login.test.jsx

```diff
@@ -0,0 +1,83 @@
+import {
+  describe,
+  it,
+  expect,
+  beforeAll,
+  beforeEach,
+  afterEach,
+  vi,
+} from "vitest";
+import { screen, waitFor } from "@testing-library/react";
+import { renderWithUser } from "../lib/utils";
+
+import Login from "./Login";
+describe("Integration | Component | Login", () => {
+  const getItemSpy = vi.spyOn(Storage.prototype, "getItem");
+  const setItemSpy = vi.spyOn(Storage.prototype, "setItem");
+
+  let originalFetch;
+  const MOCK_ACCESS_TOKEN = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9";
+
+  beforeAll(() => {
+    vi.mock("next/router", () => require("next-router-mock"));
```

Comment:
This does nothing. Beware of copying and pasting code that you don't understand.